src/components/Pagination/Pagination.stories.tsx

```diff
@@ -1,20 +1,78 @@
 import * as React from "react";
 import { action } from "@storybook/addon-actions";
+import { withDesign } from "storybook-addon-designs";
 
+import Button from "../Button/Button";
+import { ButtonTypes, ButtonIconPositions } from "../Button/ButtonTypes";
+import Icon from "../Icons/Icon";
+import { IconNames, IconRotationTypes } from "../Icons/IconTypes";
+import Label from "../Label/Label";
 import Pagination from "./Pagination";
+import Select from "../Select/Select";
 
 export default {
     title: "Pagination",
     component: Pagination,
+    decorators: [withDesign],
 };
 
 export const pagination = () => (
     <Pagination
```

Comment:
I'm like "I can't believe we didn't do this from the start" every time I see this